oaknational/Oak-Web-Application
# Chore/remove beta page

## Description

- removes beta page and beta tests

## Issue(s)

Fixes #1306 

## How to test

1. Go to /beta - it's should be gone!
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
